src/test/java/seedu/address/model/recipe/CompletionTimeTest.java

```diff
@@ -34,14 +34,19 @@ public void toString_test() {
     }
 
     @Test
-    public void equalityCheck() {
+    public void equals_test() {
         CompletionTime c1Copy = new CompletionTime(c1.value);
 
         // Test CompletionTime::equals
         assertNotEquals(c1, c1.value);
         assertEquals(c1, c1);
         assertEquals(c1, c1Copy);
         assertNotEquals(c1, c2);
+    }
+
+    @Test
+    public void hashCode_test() {
```

Comment:
Maybe this one could be something like
```suggestion
    public void test_equalityCheckForHashCodeOfTheSameObject() {
```